.travis.yml

```diff
@@ -22,9 +22,13 @@ stages:
   - name: publish
     if: type != pull_request
 
-# buildQuick needs following env (is that propagated to stages?)
-# - PRIVATE_REPO_PASS, integrationRepoUrl,
-# computed: SBT_CMD sbtArgs SCALA_VER updatedModuleVersions
+# env available in each stage
+#  - by travis config (see below): secret env vars
+#  - by `common` script: WORKSPACE, SBT_CMD, integrationRepoUrl
```

Comment:
Needs update for sbtCmd

Reply:
In `common` we have `SBT_CMD=${sbtCmd-sbt}` (this didn't change in this PR). What I understand is that `sbtCmd` can be used to override the default `SBT_CMD=sbt`. So I changed `admin/init.sh` to define `sbtCmd` instead of `SBT_CMD`.